dts/arm/st/u3/stm32u3.dtsi

```diff
@@ -231,6 +231,14 @@
 			interrupts = <66 0>;
 			status = "disabled";
 		};
+
+		dac1: dac@42028400 {
+			compatible = "st,stm32-dac";
+			reg = <0x42028400 0x400>;
+			clocks = <&rcc STM32_CLOCK(AHB2, 11)>;
+			status = "disabled";
```

Comment:
prefer `status = "disabled"` as the last property in the node for SoC DTSI files.